aicsimageio/readers/bfio_reader.py

```diff
@@ -0,0 +1,258 @@
+            ome=self._rdr.metadata,
+            scene_index=0,
+        )
+
+        coords = {d: coords[d] for d in self.out_dim_order if d in coords}
+        image_data = transforms.reshape_data(
+            image_data, self.native_dim_order, "".join(self.out_dim_order)
+        )
+
+        attrs = {constants.METADATA_PROCESSED: self._rdr.metadata}
+
+        if tiff_tags is not None:
+            attrs[constants.METADATA_UNPROCESSED] = tiff_tags
+
+        return xr.DataArray(
+            image_data,
+            dims=self.out_dim_order,
+            coords=coords,  # type: ignore
+            attrs=attrs,
+        )
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            # with fs.open(path) as open_resource:
+            with BioReader(path, backend="python"):
+
+                return True
+
+        # tifffile exceptions
+        except (TypeError, ValueError, TiffFileError):
+            return False
+
+    def __init__(
+        self,
+        image: types.PathLike,
+        chunk_dims: Optional[Union[str, List[str]]] = None,
+        out_order: str = DEFAULT_DIMENSION_ORDER,
+        **kwargs: Any,
+    ):
+        # Expand details of provided image
+        self._fs, self._path = io_utils.pathlike_to_fs(image, enforce_exists=True)
+
+        try:
+            self._rdr = BioReader(self._path, backend=self.backend)
+        except (TypeError, ValueError, TiffFileError):
+            raise exceptions.UnsupportedFileFormatError(
+                self.__class__.__name__, self._path
+            )
+
+        # Add ndim attribute so _rdr can be passed directly to dask
+        self._rdr.ndim = len(self._rdr.shape)
+
+        # Setup dimension ordering
+        dims = "YXZCT"
+        self.native_dim_order = dims[: len(self._rdr.shape)]
+        assert all(d in out_order for d in dims)
+        self.out_dim_order = [d for d in out_order if d in dims]
+
+        # Store chunking dimensions for dask, even though it shouldn't matter for bfio
+        if chunk_dims is not None:
+            log.warning(
+                "OmeTiledTiffReader does not currently support custom chunking."
+            )
+
+    @property
+    def scenes(self) -> Tuple[str, ...]:
+        return tuple(image_meta.id for image_meta in self._rdr.metadata.images)
+
+    @property
+    def current_scene(self) -> str:
+        return self.scenes[self._current_scene_index]
+
+    @property
+    def current_scene_index(self) -> int:
+        return self._current_scene_index
+
+    def set_scene(self, scene_id: Union[str, int]) -> None:
+        """
+        For all BfioReader subclasses, the only allowed value is the name of the first
+        scene since only the first scene can be read by BioReader objects. This method
+        exists primarily to help this Reader fit into existing unit test templates and
+        in case BioReader is updated to support multiple scenes.
```

Comment:
you could probably use the base class implementation if you just set the array of allowed scenes to a list of length 1.  Might eliminate the need for this function here.

Reply:
On second thought, you have to weigh that against having the reader report the true set of scenes that it finds.

Reply:
I hadn't thought of just using the base classes implementation. The main reason I did this was for documentation purposes. I think I'm going to keep it as is just for documentation reasons.

Reply:
@JacksonMaxfield and I were discussing this yesterday.  We talked about 3 different possibilities:
1. if the reader can't handle multiple scenes, and it sees a multi-scene file, then it should report the file as unsupported so that the next tiff reader implementation can take over.  That way all the data in the file will actually be available.
2. only ever report that one scene is present (in which case you can use base class implementation)
3. implement multiple scene support (in which case you can use base class implementation)
or
4. keep it as is